egs/iam/v2/local/gen_topo.py

```diff
@@ -68,23 +90,27 @@
 print("<ForPhones>")
 print(" ".join([str(x) for x in silence_phones]))
 print("</ForPhones>")
-if(args.num_sil_states > 1):
+if args.num_sil_states > 1:
     transp = 1.0 / (args.num_sil_states - 1)
-    
+
     state_str = "<State> 0 <PdfClass> 0 "
     for x in range(0, (args.num_sil_states - 1)):
-        state_str = "{}<Transition> {} {} ".format(state_str, x, transp))
+        state_str = "{}<Transition> {} {} ".format(state_str, x, transp)
     state_str = state_str + "</State>"
     print(state_str)
 
     for x in range(1, (args.num_sil_states - 1)):
-        state_str = "<State> {0} <PdfClass {0} ".format(x))
+        state_str = "<State> {0} <PdfClass {0} ".format(x)
         for y in range(1, args.num_sil_states):
-            state_str = "{}<Transition> {} {} ".format(state_str, y, transp))
+            state_str = "{}<Transition> {} {} ".format(state_str, y, transp)
```

Comment:
Here is one extra `)`. Other changes just follow the [black](https://github.com/psf/black) format style.